packages/app/src/app/components/Create/CreateBox.tsx

```diff
@@ -63,6 +83,11 @@ export const CreateBox: React.FC<CreateBoxProps> = ({
     initialCollectionId
   );
 
+  const parsedUrl = new URL(window.location.href);
+  const sandboxIdToFork =
+    sandboxIdToForkProp ?? parsedUrl.searchParams.get('sandbox');
```

Comment:
is `sandbox=<id>` passed only when the components mounts on the `<Standalone>` page? I would suggest we parse the URL in that page and pass `sandboxIdToFork` as a prop to this component, so that it behaves similarly in the two scenarios:
- dashboard URL with `create_sandbox=<id>' query param -> opens the component with the id as a prop
- standalone page with `sandbox=<id>` -> opens the component with the id as a prop

Does that make sense?

Reply:
Good one, will change it!